src/components/Search/styles.scss

```
@@ -0,0 +1,34 @@
+@import "../../styles/main.scss";
```

Single quotes in `.scss` 🤓

```suggestion
@import '../../styles/main.scss';
```